build.gradle

```diff
@@ -0,0 +1,38 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'eu.appsatori:gradle-fatjar-plugin:0.3'
+    }
+}
+
+plugins{
+    id 'java'
+    id 'eclipse'
+    id 'com.github.johnrengelman.shadow' version '1.2.3'
+    id 'maven'
+   
+}
+
```

Comment:
add thes rows in build.gradle
apply plugin: 'java'
apply plugin: 'eclipse'

Users can then create eclipse files by running ./gradlew eclipse and the import the eclipse project